src/server/scripts/Outland/Auchindoun/SethekkHalls/boss_anzu.cpp

```diff
@@ -142,7 +181,78 @@ struct boss_anzu : public BossAI
     }
 };
 
+struct npc_anzu_spirit : public ScriptedAI
+{
+    npc_anzu_spirit(Creature* creature) : ScriptedAI(creature) { }
+
+    std::array<uint32, MAX_DRUID_SPELLS> const druidSpells =
+    {
+      774, 1058, 1430, 2090, 2091, 3627, 8910, 9839, 9840, 9841, 25299, 26981, 26982, 48440, 48441, /* Rejuvenation */
+      8936, 8938, 8939, 8940, 8941, 9750, 9856, 9857, 9858, 26980, 48442, 48443, /* Regrowth */
+    };
+
+    bool HasDruidHot()
+    {
+        for (uint32 spellId : druidSpells)
+        {
+            if (me->HasAura(spellId))
+                return true;
+        }
+        return false;
+    }
```

Comment:
I don't know if it would be better... But you could check for SPELLFAMILY_DRUID + SpellIconID instead of specifying every spell?

Reply:
or SPELLFAMILY_DRUID + SPELL_AURA_PERIODIC_HEAL

Reply:
Ok, can't find it. I copied some functions that tried to locate SpellEffects using the family and effects, but none seemed to work